cmake/Modules/FindPhpConfig.cmake

```diff
@@ -0,0 +1,137 @@
+include(FindPackageHandleStandardArgs)
```

Comment:
This `FindPhpConfig.cmake` was copied from the Datadog/dd-prof-php repository. While it provides more than what we currently need, I didn't feel the need to remove anything; let me know if you disagree.